cc_unitsearch.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import json
+
+from PyQt5.QtWidgets import QApplication, QWidget, QLineEdit, QTableView, QHeaderView, QVBoxLayout
+from PyQt5.QtCore import Qt, QSortFilterProxyModel
+from PyQt5.QtGui import QStandardItemModel, QStandardItem
+
+def pullFromJson():
+    with open('CC_UNITS.json') as json_file:
+        data = json.load(json_file)
+        
+        return data
+
+class CCSearch(QWidget):
+    def __init__(self):
+        super().__init__()
+
+        self.resize(1280, 720)
+        mainLayout = QVBoxLayout()
+        units = pullFromJson()
+
+        model = QStandardItemModel(len(units), 3)
```

Comment:
Seeing how `model` is used throughout this function, it should be extracted into a class variable... is that a thing in Python though :)

Especially if we're going to separate this into several functions

Reply:
Based on Discord discussion: overkill. Pretty clear what this is.

cc @DeathWish99